styles.css

```diff
@@ -427,3 +433,196 @@ h4 {
   position: relative;
   left: 30%;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    display: contents;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar .desktop-menu-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    gap: 46px;
+    padding-right: 43px;
+    color: #ff195e;
+  }
+
+  .desktop-menu-bar a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .intro {
+    width: 337px;
+    height: 150px;
+    padding-top: 51px;
+    font-weight: 700;
+    font-size: 40px;
+    color: #172b4d;
+    letter-spacing: 0.37px;
+    padding-left: 150px;
+  }
+
+  .about-me {
+    width: 920px;
+    padding-left: 150px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    padding-top: -10px;
+    position: relative;
+    top: -40px;
+  }
+
+  .connect {
+    padding-left: 150px;
+    padding-top: 10px;
+    position: relative;
+    top: -50px;
+  }
+
+  .buttons {
+    padding-left: 150px;
+    position: relative;
+    top: -90px;
+  }
+
+  .first-card {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 40px;
+    height: 407px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 1000px;
+  }
+
+  .my-project {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    height: 80px;
+    width: 372px;
+  }
+
+  .card-img {
+    width: 54%;
+    display: flex;
+
+    /* justify-content: center; */
+  }
+
+  .descreption p {
+    font-size: 13px;
```

Comment:
- [ ] Please see the font-size for the project card's paragraph is 16px in the original design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213234718-ab535437-1190-4d90-81f1-1e384f746039.png)

As one of the goal for the project is to parse a figma template a close to as possible, please fix this :pray: